tests/foreman/ui/test_host.py

```diff
@@ -1890,7 +1889,8 @@ def test_positive_gce_provision_end_to_end(
 
     :CaseLevel: System
     """
-    name = gen_string('alpha').lower()
+    name_gen = gen_string('alpha').lower()
+    name = f'test{name_gen}'
     hostname = f'{name}.{gce_domain.name}'
```

Comment:
The `name` variable is being used at a single place, so no need to define it separately.
```suggestion
    hostname = f'test{name_gen}.{gce_domain.name}'
```

Reply:
thanks for the suggestion, I changed it so it now make more sense.